servicebus-pn-ar/README.md

```diff
@@ -0,0 +1,46 @@
+# Create a Service Bus Premium Namespace and AuthorizationRule
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fkafka-on-ubuntu%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fkafka-on-ubuntu%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+How to Run the scripts
+----------------------
```

Comment:
This section isn't needed. There's an article on how to run these on Azure.com that can be used for this.  Instead in the readme you should add text that explains the scenario that this template accomplishes